EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -32,6 +32,14 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(emptyArray.get(at: -3...(-1)), [])
         XCTAssertEqual(emptyArray.get(at: -1...1), [])
     }
+    
+    func testSafeIndex() {
+        
+        let optionalNumber: Int? = 3
+        XCTAssertEqual(numberArray[safe: 3], optionalNumber)
+        XCTAssertNotEqual(numberArray[safe: 4], 3)
```

Comment:
Is there an implicit collision safety compared to the standard array subscript impl (e.g. numberArray[3] vs numberArray[safe: 3]). If so is the compile time check on the arg name (in this case safe?)